api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -141,6 +157,28 @@ public void setUserNameClaim(String userNameClaim) {
         this.userNameClaim = userNameClaim;
     }
 
+    @Description("The fallback username claim to be used for user id if the claim specified by `userNameClaim` is not present. " +
+            "Useful when `client_credentials` authentication only results in client id being provided in another claim. " +
```

Comment:
Not sure of the possible outcomes here: 'only results in the client if being provided in another claim'

Reply:
Maybe: ''Useful when `client_credentials` authentication provides the client id in another claim, but provides no suggested username."

Reply:
> Maybe: ''Useful when `client_credentials` authentication provides the client id in another claim, but provides no suggested username."

@PaulRMellor WDYT about this suggestion?

Reply:
@PaulRMellor test